.gitignore

```diff
@@ -0,0 +1,5 @@
+/.idea/
+/.vagrant/
+/vendor/
+/build/logs/
+/composer.lock
```

Comment:
composer.lock need to deployed because composer install is used in deployment.

Reply:
Not in this case. This is a library and will not be deployed itself. Libraries should not have `composer.lock` because it could cause install conflicts between separately updated `composer.lock` files requiring the same dependency.